lib/OpenQA/BuildResults.pm

```diff
@@ -59,8 +59,9 @@ sub count_job {
             return;
         }
         if (grep { $job->result eq $_ } OpenQA::Jobs::Constants::NOT_OK_RESULTS) {
+            my $comment_data = $labels->{$job->id};
             $jr->{failed}++;
-            $jr->{labeled}++ if $labels->{$job->id};
+            $jr->{labeled}++ if $comment_data && ($comment_data->{bugs} || $comment_data->{label});
```

Comment:
even if it may look simple I think we should not have a logic like "bugs || label" in here. Maybe we need to define how we call "bugs || label" but anyway please extract in a function and use it here ...